cli/lib.rs

```diff
@@ -388,9 +388,6 @@ async fn test_command(
 }
 
 pub fn main() {
-  #[cfg(windows)]
-  ansi_term::enable_ansi_support().ok(); // For Windows 10
```

Comment:
We may need to replace this call with an equivalent in termcolor (or winapi?) ... but I don't have a Windows machine to test. Also seems like an easy fix. 

@piscisaureus I'm landing this patch. Please test colors on a recent build and see if it's still working.

Reply:
Colors are broken on Windows after this commit:
```
[0m[38;5;10mCompile file:///C:/Users/Nikolai/deno/foo.ts
[0m[1m[31merror[0m[1m TS7006: Parameter 'foo' implicitly has an 'any' type.

► [0m[38;5;14mfile:///C:/Users/Nikolai/deno/foo.ts:[0m[38;5;11m1:[0m[38;5;11m17

[0m[30m[47m1 function define(foo) {}
[0m[30m[47m  [0m[31m                ~~~
```

Reply:
@seishun Ah thanks for checking. I wonder if you could do a bit of debugging?

Here in the termcolor crate, it calls something similar to `ansi_term::enable_ansi_support()`. Is this call never reached in windows? If you set a breakpoint there do you reach it?
https://docs.rs/termcolor/1.1.0/src/termcolor/lib.rs.html#902

Reply:
It seems this function isn't even compiled. As far as I can tell, termcolor isn't used to write to stdout/stderr, so it makes sense. I guess deno will have to call set_virtual_terminal_processing directly.

Reply:
Now there's something wrong with the colors.
![image](https://user-images.githubusercontent.com/988441/75233584-bda17080-57c1-11ea-8427-002ff95894a3.png)
It looks like they're not reset properly. Is the behavior different on Linux?